helm-charts/chatqna/gaudi-values.yaml

```diff
@@ -35,7 +35,7 @@ tgi:
   # LLM_MODEL_ID: /data/OpenCodeInterpreter-DS-6.7B
   image:
     repository: ghcr.io/huggingface/tgi-gaudi
-    tag: "1.2.1"
+    tag: "2.0.1"
```

Comment:
See https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/xeon/docker_compose.yaml#L114C2-L115C63
tgi version for Xeon should also be updated to 2.1.0